### DISCUSSION TITLE
BTCGreen not showing any connections

### DISCUSSION THREAD

**Discussion by LeroyINC** [guydavis/machinaris#475] [General]
possible bug... cosmetic only

Under the connections section for BTCGreen it is showing only 2 connections - my Farmer and Wallet only

but I am able to keep the block chain and the wallet in sync and current.

when I shell into the BTCGreen docker and do a "btcgreen show -c" I see there a bunch of connections listed.
so looks like the Machinaris console is not displaying that info.

**Comment by guydavis** [comment]
Oversight on my end to not more carefully verify the Connections page for BTCGreen.  They just [released a new version](https://github.com/BTCgreen-Network/btcgreen-blockchain/releases/tag/2.2.6) to align with Chia 1.2.11. 

Testing the machinaris-btcgreen:develop image just now, this updated BTCGreen version is enough to fix the Connections page display (even in the 0.6.7 :latest version of the Machinaris main controller) image.

So, this updated btcgreen is in :develop  right now.  Will be available in :test in the next day or so.  Then will eventually be released in the next full version.  Thanks for the report.
